package.json

```diff
@@ -24,6 +24,8 @@
   },
   "dependencies": {
     "@mdi/font": "3.3.92",
+    "@sentry/browser": "5.1.1",
+    "@sentry/integrations": "^5.1.2",
```

Comment:
do not use `^` in versions, do not forget to run `yarn` when you remove the `^`

Reply:
Removed

Reply:
removed